frontend/src/pages/modelRegistry/screens/utils.ts

```
@@ -137,44 +138,42 @@ export const sortModelVersionsByCreateTime = (registeredModels: ModelVersion[]):
-      }
-      case SearchType.OWNER: {
-        return rm.owner && rm.owner.toLowerCase().includes(searchLower);
-      }
-
-      default:
-        return true;
+    const doesNotMatchModel =
+      keywordFilter &&
+      !(
+        rm.name.toLowerCase().includes(keywordFilter) ||
+        (rm.description && rm.description.toLowerCase().includes(keywordFilter)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)